src/OpenTelemetry.AutoInstrumentation/Instrumentations/MySqlData/MySqlConnectionStringBuilderIntegration.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="MySqlConnectionStringBuilderIntegration.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if !NETFRAMEWORK
```

Comment:
Will it not cause a crash on .NET Framework during reJIT? Or will not log errors?

Reply:
Good catch. It potentially could creach .NET Framework. I will create separate issue later, as I have found same issue for LoggerBuilder.

For now, fixed in dc6006f7e1ba7dcdb9b28fbb58ea33da19596091